workers/unity/Packages/io.improbable.worker.sdk/IOStream.cs

```diff
@@ -4,45 +4,67 @@
 
 namespace Improbable.Worker.CInterop
 {
+    [Flags]
     public enum OpenMode
```

Comment:
Isn't this a duplicate from `OpenMode` in `workers/unity/Packages/io.improbable.worker.sdk/CIO.cs` ?

Reply:
It is, I think I discussed this with Jamie that we shouldn't provide access to the internal Enum in `CIO.cs` and just have the same enum and just convert between the inner and outer enums